geode-core/src/main/java/org/apache/geode/cache/configuration/RegionConfig.java

```diff
@@ -148,7 +149,7 @@
 @XmlType(name = "region-type", namespace = "http://geode.apache.org/schema/cache",
     propOrder = {"regionAttributes", "indexes", "entries", "regionElements", "regions"})
 @Experimental
-public class RegionConfig implements CacheElement {
+public class RegionConfig implements CacheElement, Serializable {
```

Comment:
CacheElement itself is already Serializable.

Reply:
Sure, will fix.